src/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
```

Comment:
use 2 whitespaces instead of 4

Reply:
Done: 
![image](https://user-images.githubusercontent.com/32244670/61871236-f5250300-aee8-11e9-9cbe-aeb6fa44b90c.png)